### PR TITLE
Fix new issue notification

### PR DESCRIPTION
**Pull request by tupaschoal**
### Changes in this PR
- Sorts the languages on readme
- Sorts the usernames on the template to match the language order on readme.
- Adds everyone that is listed as a maintainer on the issue template

### Background
Before watching the repo, I thought all it took to be notified was being added as a maintainer in the README. Maybe some people are counting on it, so I'm syncing both sides.

### To everyone notified by this PR:
- If you have been notified lately, and will continue to maintain a language, you don't need to do anything.
- If you haven't been notified in a while, but still want maintain a language, you might want to check the issues on your corresponding language to make sure it's up to date. We should have been notifying you but failed, sorry.
- If you haven't been notified in a while, and don't want maintain a language anymore, let us know so we take your name off and sorry for bothering.

@Omaranwa @J0anJosep @octaroot @LPSGizmo @danidoedel @Wuzzy2 @Wirlie @frenchiveruti @JoelTroch @anon569 @LucaRed @AaronVanGeffen @telk5093 @Goddesen @Gymnasiast @renansimoes @Tupaschoal @Nubbie @BobbyS38 @izhangfei @daihakken

### CODE REVIEW

**Comment by frenchiveruti** [comment]
Hello Tulio, I wanted to keep the maintenance of the language es-ES, but I was planning on waiting until you guys sorted out how the translation files where going to be managed, what discouraged me of keeping up with the translations was the fact that some translations are on the main language file and some others are randomly dropped around in different files.  
I understand if that's a development choice, but this may change in the future so I was planning on waiting until a later date in order to help more actively.  
I'd like to keep the notifications but at the same time I understand if you need to sort out who's translating and who's not.

**Comment by JoelTroch** [comment]
I'm on the same page as @frenchiveruti for `fr-FR`, I can keep working on the main translation file but working on objects translations is gonna be too much for me.

**Comment by AaronVanGeffen** [comment]
> what discouraged me of keeping up with the translations was the fact that some translations are on the main language file and some others are randomly dropped around in different files.  

I ran into the same problem. For this reason, I added two Python scripts to the root of the objects repository. Running the first, you can extract all object strings for a particular language to just one file. This makes it much easier to translate. The second one allows you to do the reverse.

This is by no means a perfect solution, but hopefully it helps.

**Comment by Gymnasiast** [comment]
@JoelTroch 
> I can keep working on the main translation file but working on objects translations is gonna be too much for me.

Hm, what seems the be the issue?

**Comment by AaronVanGeffen** [comment]
@Gymnasiast Having to translate all object files separately is, it seems, and understandably so. Particularly when translating exclusively through GitHub, this seems like quite a pain.

**Comment by Gymnasiast** [comment]
@AaronVanGeffen I won't deny that, but I wanted to hear from Joël. He might have his own reasons why it doesn't work for him.

**Comment by JoelTroch** [comment]
@Gymnasiast Basically what @AaronVanGeffen said, except that I don't use GitHub's web IDE/editor or whatever you call that directly

File: README.md [hunk 1]
```diff
@@ -42,19 +42,19 @@ One simple way to test the new language is to copy the whole document you've tra
...
-| [![](https://img.shields.io/badge/de--DE-maintained-green.svg)](https://github.com/OpenRCT2/Localisation/blob/master/data/language/de-DE.txt) | [danidoedel](https://github.com/danidoedel)        |
+| [![](https://img.shields.io/badge/da--DK-maintained-green.svg)](https://github.com/OpenRCT2/Localisation/blob/master/data/language/da-DK.txt) | [LPSGizmo](https://github.com/LPSGizmo)          |
+| [![](https://img.shields.io/badge/de--DE-maintained-green.svg)](https://github.com/OpenRCT2/Localisation/blob/master/data/language/de-DE.txt) | [danidoedel](https://github.com/danidoedel), [Wuzzy2](https://github.com/Wuzzy2)        |
+| [![](https://img.shields.io/badge/en--GB-maintained-green.svg)](https://github.com/OpenRCT2/OpenRCT2/blob/develop/data/language/en-GB.txt   ) | -Anyone-                                           |
 | [![](https://img.shields.io/badge/es--ES-maintained-green.svg)](https://github.com/OpenRCT2/Localisation/blob/master/data/language/es-ES.txt) | [Wirlie](https://github.com/Wirlie), [frenchiveruti](https://github.com/frenchiveruti)                |
```
Comment (Gymnasiast) [inline README.md:50]: Remove frenchiveruti here.

Comment (tupaschoal) [reply README.md]: done

**Pull request merged**